package/kedro_viz/models/graph.py

```diff
@@ -403,18 +408,21 @@ class TaskNodeMetadata(GraphNodeMetadata):
 
     def __post_init__(self, task_node: TaskNode):
         kedro_node = cast(KedroNode, task_node.kedro_obj)
-        self.code = inspect.getsource(
-            _extract_wrapped_func(cast(FunctionType, kedro_node._func))
-        )
-        code_full_path = Path(inspect.getfile(kedro_node._func)).expanduser().resolve()
-        try:
-            filepath = code_full_path.relative_to(Path.cwd().parent)
-        except ValueError:  # pragma: no cover
-            # if the filepath can't be resolved relative to the current directory,
-            # e.g. either during tests or during launching development server
-            # outside of a Kedro project, simply return the fullpath to the file.
-            filepath = code_full_path
-        self.filepath = str(filepath)
+        if inspect.isfunction(kedro_node.func):

```

Comment:
Could you put a comment here stating that this is needed because `kedro_node.func` could be partial or curried functions?